components/collapsible-panel/collapsible-panel-group.js

```diff
@@ -0,0 +1,36 @@
+import { css, html, LitElement } from 'lit';
+import { classMap } from 'lit/directives/class-map.js';
+import { SkeletonMixin } from '../skeleton/skeleton-mixin.js';
+
+/**
+ * A component that renders collapsible panels
+ * @slot default - Slot for panels. Only accepts `d2l-collapsible-panel`
+ */
+class CollapsiblePanelGroup extends SkeletonMixin(LitElement) {
+
+	static get styles() {
+		return [super.styles, css`
+			:host ::slotted(*) {
+				display: none
+			}
+			:host ::slotted(d2l-collapsible-panel) {
+				display: unset;
+			}
+			:host .padding {
+				display: flex;
+				flex-direction: column;
+				row-gap: 0.5rem;
+			}
+		`];
+	}
+
+	render() {
+		const classes = {
+			padding: this.children?.[0].type !== 'inline'
+		};
```

Comment:
Not crazy about this. An alternative would be to have `d2l-collapsible-panel` handle spacing itself with a `margin-bottom`